src/app/layout/NavigationDrawer/NavigationDrawer.js

```diff
@@ -48,47 +48,47 @@ export default function TemporaryDrawer() {
       <List>
-        <ListItem button>
-          <ListItemIcon>
-            <AssignmentIcon classes={{ root: classes.colorIcon }} fontSize="large" />
-          </ListItemIcon>
-          <Link to="/missions" className={classes.link}>
-            <ListItemText primary="Volunteer needed" />
-          </Link>
-        </ListItem>
-        <PrivateComponent>
-          <ListItem button>
-            <ListItemIcon>
-              <AccountCircleIcon classes={{ root: classes.colorIcon }} fontSize="large" />
-            </ListItemIcon>
-            <Link to="/user/profile" className={classes.link}>
-              <ListItemText primary="User Profile" />
-            </Link>
-          </ListItem>
+        <Link to="/missions" className={classes.link}>
           <ListItem button>
             <ListItemIcon>
               <AssignmentIcon classes={{ root: classes.colorIcon }} fontSize="large" />
             </ListItemIcon>
-            <Link to="/missions/volunteered" className={classes.link}>
-              <ListItemText primary="Volunteered Missions" />
-            </Link>
+            <ListItemText primary="Volunteer needed" />
           </ListItem>
-          <ListItem button>
-            <ListItemIcon>
-              <AssignmentIcon classes={{ root: classes.colorIcon }} fontSize="large" />
-            </ListItemIcon>
-            <Link to="/missions/created" className={classes.link}>
+        </Link>
+        <PrivateComponent>
+          <Link to="/user/profile" className={classes.link}>
+            <ListItem button>
+              <ListItemIcon>
+                <AccountCircleIcon classes={{ root: classes.colorIcon }} fontSize="large" />
+              </ListItemIcon>
+              <ListItemText primary="User Profile" />
+            </ListItem>
+          </Link>
+          <Link to="/missions/volunteered" className={classes.link}>
+            <ListItem button>
+              <ListItemIcon>
+                <AssignmentIcon classes={{ root: classes.colorIcon }} fontSize="large" />
+              </ListItemIcon>
+              <ListItemText primary="Volunteered Missions" />
+            </ListItem>
+          </Link>
+          <Link to="/missions/created" className={classes.link}>
+            <ListItem button>
+              <ListItemIcon>
+                <AssignmentIcon classes={{ root: classes.colorIcon }} fontSize="large" />
+              </ListItemIcon>
               <ListItemText primary="My Requests" />
-            </Link>
-          </ListItem>
-          <ListItem button>
-            <ListItemIcon>
-              <EmojiPeopleIcon classes={{ root: classes.colorIcon }} fontSize="large" />
-            </ListItemIcon>
-            <Link to="/status" className={classes.link}>
+            </ListItem>
+          </Link>
+          <Link to="/status" className={classes.link}>
+            <ListItem button>
+              <ListItemIcon>
+                <EmojiPeopleIcon classes={{ root: classes.colorIcon }} fontSize="large" />
```

Comment:
Can we add titleAccess prop for accessibility issues?

Reply:
I saw some changes that were made to the design of the drawer that Tuan posted https://mutualaidworld.slack.com/files/UVD8UBX0B/F011QQ3A96G/image.png

Do you want to wait to see if the end product plans on removing Icons from this component all together?

Reply:
We can merge this in and then remove the icons later if we want. There are some other PRs waiting for this one to get in, so that they can resolve conflicts merging this one might cause.

Reply:
Partly what i was suggesting. If we're going to remove the Icons anyways, do we need to make them more accessible now?

Reply:
Nope. You are right